ivy/functional/frontends/mindspore/ops/function/nn_func.py

```diff
@@ -36,3 +36,8 @@ def selu(input_x):
 @to_ivy_arrays_and_back
 def softsign(x):
     return ivy.divide(x, ivy.add(1, ivy.abs(x)))
+
+@with_supported_dtypes({"2.0.0 and below": ("int8", "int16", "int32", "int64", "float16", "float32", "float64")}, "mindspore")
+@to_ivy_arrays_and_back
+def adaptive_maxpool2d(input, output_size):
```

Comment:
We need to take care of the `return_indices=False` parameter in the original native function as well: https://www.mindspore.cn/docs/en/r2.1/api_python/ops/mindspore.ops.adaptive_max_pool2d.html?highlight=adaptive#mindspore.ops.adaptive_max_pool2d